tests/integration_tests/default_value/config.toml

```diff
@@ -0,0 +1,21 @@
+# Importer Configuration.
```

Comment:
Seems this file can be removed now.

Reply:
In run.sh, all these two toml files are used.

Reply:
All right, please refine the comment, it was copied from binlog repo originally, `importer configuration` is misleading